.doc_gen/templates/zonbook/service_chapter_template.xml

```diff
@@ -34,13 +58,33 @@
     {{- with $cat_examples := index $.CategorizedExampleSets $category}}
     <section id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}" role="topic">
         <info>
-            <title id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.title">{{$cat_examples.CategoryName}} examples for {{$.ServiceEntity.Short}}</title>
-            <titleabbrev id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.titleabbrev">{{$cat_examples.CategoryName}} examples</titleabbrev>
+            <title id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.title">{{$cat_examples.CategoryName}} for {{$.ServiceEntity.Short}}</title>
+            <titleabbrev id="{{$doc_id}}_{{$cat_examples.CategoryIdSuffix}}.titleabbrev">{{$cat_examples.CategoryName}}</titleabbrev>
             <abstract>
                 <para>The following code examples show how to use {{$.ServiceEntity.Short}} with &AWS; SDKs.</para>
             </abstract>
         </info>
+        {{- if eq $cat_examples.CategoryName "Cross-service examples"}}
+        <para>The following sample applications use &AWS; SDKs to combine {{$.ServiceEntity.Short}}
+            with other &AWS; services. Each example includes a link to GitHub where you can find
+            instructions on how to set up and run the application.</para>
```

Comment:
<para>The following sample applications use &AWS; SDKs to combine {{$.ServiceEntity.Short}}
            with other &AWS; services. Each example includes a link to GitHub, where you can find
            instructions on how to set up and run the application.</para>

Reply:
Done